library/core/src/fmt/mod.rs

```diff
@@ -703,11 +703,21 @@ pub use macros::Debug;
 
 /// Format trait for an empty format, `{}`.
 ///
-/// `Display` is similar to [`Debug`], but `Display` is for user-facing
-/// output, and so cannot be derived.
+/// `Display` is similar to [`Debug`], but `Display` is for user-facing output,
+/// and so cannot be derived. When `Display` is implemented, the [`ToString`]
+/// trait is implemented automatically, adding the `to_string` method to all
+/// `Display` types.
+///
+/// A `Display` implementation should be lossless and infallible, otherwise it
```

Comment:
How do you define lossless here? Any definition I can think of feels way too strict..

Reply:
"doesn't lose information from its internal data" and then we can use `Path` as an example?